docs/sdk-and-tools/indices/accountsesdt.md

```diff
@@ -44,9 +44,8 @@ curl --request GET \
   --data '{
     "query": {
         "match": {
-         "identifier": {
-           "query": "MY-TOKEN-aaabbb",
-           "operator": "AND"
+         "token": {
+           "query": "MY-TOKEN-aaabbb"
```

Comment:
correct. but please add `"operator": "AND"` as well since its absence can cause more results than needed